packages/nextjs/components/Header.tsx

```diff
@@ -20,14 +56,65 @@ const NavLink = ({ href, children }: { href: string; children: React.ReactNode }
   );
 };
 
+const NavLinks = ({ setIsOpen }: { setIsOpen: React.Dispatch<React.SetStateAction<boolean>> }) => {
+  return (
+    <>
+      {menuItems.map((item, index) => {
+        return (
+          <li key={index}>
+            <NavLink href={item.link} setIsOpen={setIsOpen}>
+              {item.icon}
+              {item.label}
+            </NavLink>
+          </li>
+        );
+      })}
+    </>
+  );
+};
+
+/* End of possible Navlinks Components */
+
 /**
  * Site header
  */
 export default function Header() {
+  const [isOpen, setIsOpen] = useState(Boolean);
+  const ref = useRef<HTMLInputElement>(null);
```

Comment:
Instead of `HTMLInputElement ` we can use `HTMLDivElement` since we are keeping reference of `Div`